GitVersionCore/VersioningModes/VersioningMode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace GitVersion
+{
+    public enum VersioningMode
```

Comment:
Any thoughts on a better name for this?

Continuous Deployment causes a version change every commit, continuous delivery will keep same semver until deployment/tag then bump.